### PR TITLE
Added seeding for all examples

### PR DESCRIPTION
**Pull request by Tsytrytskyi**
Summary
---------

Add more seed content to the database.

Test Plan
---------

```
flask seed
```

### CODE REVIEW

File: open-media-match/src/OpenMediaMatch/app.py [hunk 1]
```diff
@@ -54,6 +54,10 @@ def index():
             "index.html.j2", production=app.config.get("PRODUCTION")
         )
 
+    @app.route("/hello")
+    def hello_world():
+        return "Hello, world!\n"
+
```
Comment (Dcallies) [inline open-media-match/src/OpenMediaMatch/app.py:60]: Blocking: Remove this code from the hello world example

**Review by Dcallies — CHANGES_REQUESTED** [review]
blocking: You need to run black formatting for the CI to pass.

You can configure VSCode to do it for you so you don't have to remember, will show you how.

File: open-media-match/src/OpenMediaMatch/app.py (hunk 1 above)
Comment (Tsytrytskyi) [reply open-media-match/src/OpenMediaMatch/app.py]: Done

**Comment by Dcallies** [comment]
@Tsytrytskyi showed me his passing CLA offline.

**Review by Dcallies — APPROVED** [review]
All issues addressed

**Pull request merged**